core/src/main/web/app/helpers/helper.js

```diff
@@ -100,6 +100,41 @@
           };
         }
       },
+      initBeakerLanguageSettings: function () {
+        var beakerObj = this.getBeakerObject();
+        var beaker = beakerObj.beakerObj;
+        if (beaker) {
+          beaker.language = {};
+        }
+      },
+      setLanguageManagerSettingsToBeakerObject: function (plugin) {
+        var beakerObject = this.getBeakerObject().beakerObj;
+        if (beakerObject && beakerObject.language) {
+          var spec = plugin.spec;
+          var beakerLanguageSettings = {};
+          for (var property in spec){
```

Comment:
This is fine but you might want to use lo-dash sugar for iterating own properties:
https://lodash.com/docs#forOwn

I just think it cuts down on the boilerplate, but its your choice